### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please do not use git clone or `ldd`.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+image: ghcr.io/capuanob/vedis-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit image name.

File: fuzz/fuzz.c [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#include <stdint.h> //uint8_t
```
Comment (xansec) [inline fuzz/fuzz.c]: Can we change the name of this to indicate what's being fuzzed?

**Review by xansec — CHANGES_REQUESTED** [review]
Is the top level CMakeList necessary? Wondering because it doesn't look like this project uses cmake? Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Done! Not sure why I overengineered with a CMake project in the first place... reverted to just running clang

**Pull request merged**